_integration-schemas/closeio/v1/users.md

```diff
@@ -0,0 +1,29 @@
+    - name: date_created
+      type: ['null', string]
```

Comment:
`type` should be formatted like this, if a single type: `"string"`

When there are multiple possibilities: `"string, date-time"`

`null` shouldn't be listed as a type.